storage/database/leveldb_database.go

```diff
@@ -99,6 +99,12 @@ type levelDB struct {
 	diskWriteMeter  metrics.Meter // Meter for measuring the effective amount of data written
 	blockCacheGauge metrics.Gauge // Gauge for measuring the current size of block cache
 
+	perfCheck           bool
+	getTimeMeter        metrics.Meter
+	putTimeMeter        metrics.Meter
+	batchWriteTimeMeter metrics.Meter
+	batchWriteSizeMeter metrics.Meter
```

Comment:
`batchWriteSizeMeter` is declared, but not used anywhere. Is it for matching with metrics of Dynamo?